cinema/tests/test_movie_api.py

```diff
@@ -157,3 +158,164 @@
         res = self.client.get(MOVIE_SESSION_URL)
 
         self.assertIn("movie_image", res.data[0].keys())
+
+
+class UnauthenticatedMovieAoiTest(TestCase):
+    def setUp(self):
+        self.client = APIClient()
+
+    def test_authRequired(self):
+        res = self.client.get(MOVIE_URL)
+        self.assertEqual(res.status_code, status.HTTP_401_UNAUTHORIZED)
+
+
+class AuthenticatedMovieAoiTest(TestCase):
```

Comment:
The class name `AuthenticatedMovieAoiTest` seems to have a typo. It should likely be `AuthenticatedMovieApiTest`.